src/plugins/deleteDecorator/utils/deleteDecoratorRegex.ts

```diff
@@ -0,0 +1,8 @@
+import { numbers } from 'plugins/unitDecorator/utils/unitDecoratorRegex';
+
+export const getUnitTotallyMatch = (units = '코'): RegExp =>
+  new RegExp(`^[${numbers}]+[${units}]`, 'gi');
+
+export const getAllGroupesIntoSpace = (): RegExp =>
```

Comment:
```suggestion
export const getAllGroupsIntoSpace = (): RegExp =>
```
p3: -es가 아니라 -s 아닌가요!?

Reply:
아앗,,,[반영](https://github.com/k-roffle/knitting-frontend/pull/43/commits/f0869f4fee67c847163c340b4834c3eea695ab17) 했습니다~!